docs/eventing/experimental-features.md

```diff
@@ -117,4 +139,60 @@ spec:
     timeout: PT5S
 ```
 
-You can specify a `delivery` spec for Channels, Subscriptions, Brokers, Triggers, and any other resource spec that accepts the `delivery` field.
+You can specify a `delivery` spec for Channels, Subscriptions, Brokers,
+Triggers, and any other resource spec that accepts the `delivery` field.
+
+### Knative Reference mapping
+
+**Flag name**: `kreference-mapping`
+
+**Stage**: Alpha, disabled by default
+
+**Tracking issue**: [#5148](https://github.com/knative/eventing/issues/5593)
+
+**Persona**: Administrator, Developer
+
+When enabled, this feature allows you to provide mappings from
+[Knative reference](https://github.com/knative/specs/blob/main/specs/eventing/overview.md#destination)
+to templated URI. For instance it allows you to directly reference non-addressable resources everywhere Knative Eventing accepts a reference (e.g. PingSource sink, Trigger subscriber, etc...).
+
+Mappings are defined (by the administrator) in `config-reference-mapping`. The following example maps `JobDefinition` to a Job runner service:
+
+{% raw %}
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-kreference-mapping
+  namespace: knative-eventing
+data:
+  JobDefinition.v1.mygroup:
+    "https://jobrunner.{{ .SystemNamespace }}.svc.cluster.local/{{ .Name }}"
+```
+{% endraw %}
+
+The key must be of the form `<Kind>.<version>.<group>`. The value must resolved
+to a valid URI. Currently, the following template data are supported:
+
+- Name: name of the referenced object
+- Namespace: namespace of the referenced object
+- SystemNamespace: namespace of where Knative Eventing is installed
+
+Given the above mapping, the following example shows how a developer can directly reference `JobDefinition` objects in PingSource:
+
+```yaml
+apiVersion: sources.knative.dev/v1
+kind: PingSource
+metadata:
+  name: trigger-job-every-minute
+spec:
+  schedule: "*/1 * * * *"
+  sink:
+    ref:
+      apiVersion: mygroup/v1
+      kind: JobDefinition
+      name: ajob
+```
+
+!!! note
+    Currently only PingSource supports this experimental feature.
```

Comment:
move this note way higher in this section - someone would want to know this before they enable it

Reply:
done.